sk-python-hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "cwd": "${workspaceFolder}",
+            "program": "hello_world/main.py", // or ${file} for the current file
+            "console": "integratedTerminal",
+            "justMyCode": true
+        }
+    ]
```

Comment:
We also need

1. `extensions.json` which lists recommended extensions and should include the SK extension, see: https://github.com/microsoft/semantic-kernel-starters/blob/main/sk-csharp-hello-world/.vscode/extensions.json. Some users may just clone the repo.
2. `tasks.json` which should include any build steps that need to be executed before running. You can use the `prelaunchTask` property to ensure required tasks are executed before running.

Reply:
Thanks, I didn't know about these.